src/hotspot/share/prims/jvmti.xml

```diff
@@ -15468,6 +15468,10 @@ typedef void (JNICALL *jvmtiEventVMInit)
        - Add new error code:
          - JVMTI_ERROR_UNSUPPORTED_OPERATION
   </change>
+  <change date="7 June 2023" version="21.0.0">
+      Virtual threads finalized to be a permanent feature.
+      Agent start-up in the live phase now specified to print a warning.
+  </change>
```

Comment:
This is just the history text, no normative changes, shouldn't need a CSR.

Reply:
Okay, thanks. Updated the description.